gwpy/io/ligolw.py

```diff
@@ -533,10 +533,16 @@ def write_tables(target, tables, append=False, overwrite=False, **kwargs):
 
     # write file
     if isinstance(target, str):
-        kwargs.setdefault('gz', target.endswith('.gz'))
+        kwargs.setdefault(
+            'compress',
+            "gz" if target.endswith('.gz') else False,
+        )
```

Comment:
I have a feeling that this will cause errors on systems that have `lscsoft-glue` and _not_ `python-ligo-lw` (since `glue.ligolw.utils.write_filename` is still expecting `gz`), but I would have hoped that the unit tests should have caught this.

Reply:
Also, can you please forgive me for asking you to pick one style of quotes for both lines?

Reply:
https://youtu.be/hou0lU8WMgo

Reply:
@duncanmmacleod, would you prefer that I do some sort of try-except statement?